### PR TITLE
Add endpoint for getting PMIDs for a single node

### PR DESCRIPTION
**Pull request by sarah-seitanakis**
Similar to getting shared PMIDs.

### CODE REVIEW

File: ranker/shared/omnicorp_postgres.py [hunk 1]
```diff
@@ -58,6 +58,30 @@ async def get_prefixes(self):
         prefixes = [ row['table_name'] for row in rows ]
         return prefixes
 
+    async def get_pmids(self, node):
+        """Get shared PMIDs."""
+        prefix = get_postgres_curie_prefix(node)
+        if (
```
Comment (cbizon) [inline ranker/shared/omnicorp_postgres.py:64]: Formatting wise, I think I would just do `if prefix not in self.prefixes:`

File: ranker/shared/omnicorp_postgres.py [hunk 2]
```diff
@@ -58,6 +58,30 @@ async def get_prefixes(self):
...
+                prefix not in self.prefixes
+        ):
+            return 0
```
Comment (cbizon) [inline ranker/shared/omnicorp_postgres.py:67]: Is returning 0 the right thing?  It looks like this function can either return a list of curies, an empty list, None, or 0.  I think None and 0 should be replaced with [].

Comment (cbizon) [inline ranker/shared/omnicorp_postgres.py:67]: Is returning 0 the right thing?  It looks like this function can either return a list of curies, an empty list, None, or 0.  I think None and 0 should be replaced with [].

**Comment by sarah-seitanakis** [comment]
I was unable to test these last 3 changes because I was unable to get a stable connection to the Sterling cluster, any ideas? @cbizon

**Review by cbizon — APPROVED** [review]

**Comment by sarah-seitanakis** [comment]
Tests passed! @cbizon

**Pull request merged**